### PR TITLE
Ck 42 chores page

### PR DESCRIPTION
**Pull request by Amikz**
Created a simple table and added buttons for creating a chore and viewing account details. Table chores are (temporarily) hard-coded. Log out by opening the dropdown menu and clicking log out.
image

### CODE REVIEW

**Review by JoshuaDueck — APPROVED** [review]
Overall looks pretty good! Good job! I think there will be some things we can change after the PrimeReact component library is added. It provides components like buttons and dropdowns that are styled for us.

File: frontend/src/components/pages/ChoresPage.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { React } from "react";
 import { Navigate, useNavigate } from "@tanstack/react-location";
+import { useState } from "react";
+import { useMemo } from "react";
```
Comment (Ahsen19) [inline frontend/src/components/pages/ChoresPage.js:4]: I think you can group lines 3 and 4 like:
`import { React, useState, useMemo } from "react"`

**Review by Ahsen19 — APPROVED** [review]
Just added a sugestion that might be helpful in future. Other than that this is looks awesome!

File: frontend/src/components/pages/ChoresPage.js (hunk 1 above)
Comment (Amikz) [reply frontend/src/components/pages/ChoresPage.js]: ty I made the change

**Review by Ahsen19 — APPROVED** [review]
Awesome!

**Pull request merged**